test/e2e/app-dir/revalidatetag-rsc/revalidatetag-rsc.test.ts

```diff
@@ -29,8 +29,10 @@ describe('revalidateTag-rsc', () => {
     expect(randomNumber).toEqual(randomNumber2)
 
     await browser.elementByCss('#revalidate-via-page').click()
-    await browser.waitForElementByCss('#home')
-    await browser.elementByCss('#home').click()
+    const refreshButton = await browser.waitForElementByCss('#refresh')
```

Comment:
This test was recently added in https://github.com/vercel/next.js/pull/70446 but it makes an incorrect assumption about client router cache invalidation.

It was working previously because we weren't honoring the static staletime for the static page, but since the fix in this PR, it now respects the static staletime. And as described in the [docs](https://nextjs.org/docs/app/building-your-application/caching#invalidation-1), you'd need to have called a `revalidate` API in a server action to simultaneously clear the client router cache.